data/json/items/containers.json

```diff
@@ -76,6 +76,96 @@
     "watertight": true,
     "qualities": [ [ "BOIL", 2 ], [ "CONTAIN", 1 ] ]
   },
+  {
+    "id": "hp_pump",
+    "type": "TOOL",
+    "category": "tools",
+    "name": "high pressure air pump",
+    "description": "This is a high pressure air pump.  It has an adjustable receptacle designed to accept a wide variety of refillable high pressure air cylinders.",
+    "weight": 2000,
+    "volume": 2,
+    "price": 1000,
+    "material": "steel",
+    "symbol": ":",
+    "color": "light_gray",
+    "ammo": "air",
+    "use_action": "HAND_PUMP_INTEGRAL",
+    "looks_like": "air_pump",
+    "magazines": [ [ "air", [ "heavy_air_cylinder", "medium_air_cylinder", "light_air_cylinder" ] ] ]
+  },
+  {
+    "id": "hp_hose",
+    "type": "TOOL",
+    "name": "air fitting",
+    "name_plural": "air fittings",
+    "description": "This is an air fitting and hose designed for high pressure.  It can connect a high pressure cylinder to a variety of devices utilizing air as power.",
+    "weight": 500,
+    "volume": "200 ml",
+    "price": 2800,
+    "to_hit": -1,
+    "bashing": 8,
+    "material": [ "steel" ],
+    "symbol": ";",
+    "color": "light_gray",
+    "ammo": "air",
+    "looks_like": "hose",
+    "magazines": [ [ "air", [ "heavy_air_cylinder", "medium_air_cylinder", "light_air_cylinder" ] ] ],
+    "magazine_well": 1
+  },
+  {
+    "id": "heavy_air_cylinder",
+    "type": "MAGAZINE",
+    "category": "spare_parts",
+    "name": "heavy air cylinder",
+    "name_plural": "heavy air cylinders",
```

Comment:
Redundant. Trailing `s` is automatically added to `name` field if `name_plural` field isn't defined.